test/relayclient/RelayClient.test.ts

```diff
@@ -822,6 +827,19 @@ contract('RelayClient', function (accounts) {
         gas: '0xf4240',
         gasPrice: '0x51f4d5c00'
       })
+      const commitment = new Commitment(
+        pingResponse.maxDelay,
+        from,
+        to,
+        data,
+        relayHub.address,
+        relayWorkerAddress
+      )
+      badCommitmentReceipt = {
```

Comment:
the js linter is rightly pointing out that we are currently not using this, even though a value is set

Reply:
This was solved